src/actions/make/_make.lua

```diff
@@ -178,9 +178,21 @@
 -- Format a list of values to be safely written as part of a variable assignment.
 --
 
-	function make.list(value)
+	function make.list(value, quoted)
+		quoted = false
```

Comment:
My mistake, I was testing your PR locally and checking if this change made any difference in the test suite and forgot to remove it the debug line.

@tvandijck mind removing that line from master?